### ISSUE TITLE
Stub parser not able to parse integer and long arguments in type annotations

### ISSUE THREAD

**Issue by kelloggm** [typetools/checker-framework#1542]
Consider the following pair of Java classes:

```java
import org.checkerframework.common.value.qual.IntRange;

class NeedsIntRange {
    static int range(boolean big) {
	if (big) {
	    return 20000;
	} else {
	    return 3;
	}
    }
}

class UseIntRange {
    void do_things() {
	@IntRange(from = 3, to = 20000) int x = NeedsIntRange.range(true);
    }
}
```
The first class needs an IntRange annotation in order to permit the value checker to typecheck it. Suppose it is supplied by a stub file:
```java
import org.checkerframework.common.value.qual.IntRange;
class NeedsIntRange {
      @IntRange(from = 3, to = 20000) int range(boolean big);
}
```
Then, executing these two commands results in an error in the stubparser:
```bash
javac NeedsIntRange.java
javac -processor org.checkerframework.common.value.ValueChecker -Astubs=intrange.astub NeedsIntRange.java
```
The error looks like this:
```
error: given value differs from expected; found: java.lang.Integer; expected: java.lang.Long
```
`@IntRange` does require a long, but changing the values in the stub file to longs (which shouldn't actually be required, as integers should be convertible) results in a different error:
```
error: StubParser: unhandled annotation attribute type: 3L class: class com.github.javaparser.ast.expr.LongLiteralExpr
```
With both of these errors present, there doesn't seem to be any way to write an `@IntRange` annotation in a stub file.

**Comment by wmdietl** [comment]
Fixed by https://github.com/typetools/checker-framework/commit/cfba539a5c0573920dd431b36557fd348078d9ab
